### PR TITLE
[DOCS] service state and module docs do not mention that systemd daem…

### PR DESCRIPTION
**Pull request by Tanmoy037**
…on is reloaded #63601 --solved

### What does this PR do?
adding [DOCS] "service" state and module docs do not mention that systemd daemon is reloaded #63601
### What issues does this PR fix or reference?
Fixes: The service state documentation and the systemd_service module documentation make clear that this built-in feature exists.

### Previous Behavior
Remove this section if not relevant

### New Behavior
Remove this section if not relevant

### Merge requirements satisfied?
**[NOTICE] Bug fixes or features added to Salt require tests.**

- [ *] Docs
- [ *] Changelog - https://docs.saltproject.io/en/master/topics/development/changelog.html
- [ ] Tests written/updated

### Commits signed with GPG?
Yes/No

Please review [Salt's Contributing Guide](https://docs.saltproject.io/en/master/topics/development/contributing.html) for best practices.

See GitHub's [page on GPG signing](https://help.github.com/articles/signing-commits-using-gpg/) for more information about signing commits with GPG.

### CODE REVIEW

**Comment by cmcmarrow** [comment]
Thank you @Tanmoy037 for the PR. We just need a change log and this should be good to merge. https://docs.saltproject.io/en/master/topics/development/changelog.html

Because this is just a doc change no need to write test :)

**Comment by cmcmarrow** [comment]
Thank you @Tanmoy037, I'll get it merged once master gets updated with 3006.x

File: changelog/63601.added.md [hunk 1]
```diff
@@ -0,0 +1 @@
+systemd daemon is reloaded, added in service state and module docs.
```
Comment (barbaricyawps) [inline changelog/63601.added.md:1]: FYI, we don't need changelogs for doc changes. 😄

**Review by barbaricyawps — APPROVED** [review]
This looks great! It seems well-communicated to me.

File: changelog/63601.added.md (hunk 1 above)
Comment (Tanmoy037) [reply changelog/63601.added.md:1]: @barbaricyawps Noted

**Review by s0undt3ch — APPROVED** [review]

**Comment by s0undt3ch** [comment]
Can't push fixes to your branch. Submitted in https://github.com/saltstack/salt/pull/64149

**Pull request closed without merging**